.github/workflows/ios-android-test.yml

```diff
@@ -45,7 +48,7 @@ jobs:
               uses: ReactiveCircus/android-emulator-runner@76c2bf6f95ed6458fd659a1bdb680a0f8df232dc
               if: |
                 !github.head_ref
-                && ( github.ref == 'refs/heads/release' || github.ref == 'refs/heads/main' )
+                && ( startsWith(github.ref, 'refs/heads/release') != true || github.ref == 'refs/heads/main' )
```

Comment:
Why `!= true`? Shouldn't it be `== true`?

Reply:
... true

Reply:
`== true`